apps/storybook/src/TiledHeatmapMesh.stories.tsx

```diff
@@ -216,6 +218,60 @@ FlippedAxes.args = {
   },
 };
 
+function LinearAxesGroup(props: { children: ReactNode }) {
```

Comment:
Wouldn't that make sense to have this in the lib at some point?
(I'll something like this soon)

Reply:
This is now handling position, scale and flipped axes.
What's eventually missing is that it does not "cancel" transforms applied by parents of this group (that should be possible if needed), so it's meant to be used as a "root" group.